src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -17,42 +17,102 @@ export interface StatusProps {
 }
 
 export class StatusRoute extends Component<StatusProps> {
+  stateCopy = () => {
+    const { state, requestedFulfillment } = this.props.order
+    switch (state) {
+      case "SUBMITTED":
```

Comment:
It feels like we should have a default for these switch statements. Do you think `SUBMITTED` makes sense for that?

Reply:
I had a default before, but I'm not sure... I mean, if some case gets introduced and it's not _really_ in a submitted state it doesn't seem like that would make sense. Maybe showing an error state or something.

Reply:
That's okay. We can also cross this bridge when we get there